### PR TITLE
Add Security-Kerberos Event Log for Desktop Troubleshooting

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/desktop-access/troubleshooting.mdx [hunk 1]
```diff
@@ -32,6 +32,17 @@ If the "Smart Card" service is not running, open PowerShell and run
 `gpupdate.exe /force`. This forces a Group Policy sync and should pick up the
 service changes.
 
+### Smart card not supported for Account
+
+You connect to a Windows host and get the error message: **"Signing in with a smart card isn't supported for your account."**
+or similar.
```
Comment (alexfornuto) [inline docs/pages/desktop-access/troubleshooting.mdx:38]:
```suggestion
You connect to a Windows host and get the error message: **"Signing in with a smart card isn't supported for your account."**
or similar.
```

File: docs/pages/desktop-access/troubleshooting.mdx [hunk 2]
```diff
@@ -32,6 +32,17 @@ If the "Smart Card" service is not running, open PowerShell and run
...
+### Smart card not supported for Account
+
+You connect to a Windows host and get the error message: **"Signing in with a smart card isn't supported for your account."**
+or similar. 
+
+**Solution:** Review the Security-Kerberos Log on the Windows host for causes.
+
+The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts.  This is available
+at `Event Viewer->Application and Services Logs->Microsoft->Windows->Security-Kerberos->Operational` in the Windows Event Log.  By default this
+log is not enabled. Enable and attempt to connect with Teleport UI to review log entries.
```
Comment (alexfornuto) [inline docs/pages/desktop-access/troubleshooting.mdx:44]:
```suggestion
The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts. This is available
at `Event Viewer->Application and Services Logs->Microsoft->Windows->Security-Kerberos->Operational` in the Windows Event Log. By default this
log is not enabled. Enable and attempt to connect with Teleport UI to review log entries.
```

**Review by alexfornuto — APPROVED** [review]
Removed some extraneous spaces, otherwise 👍

File: docs/pages/desktop-access/troubleshooting.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/desktop-access/troubleshooting.mdx:44]: Does anyone else find the flow a bit awkward here?

It's not enabled by default, but the order in which we describe the solution is 1) check the logs, and 2) enable the logs. Shouldn't it be the other way around?

Comment (alexfornuto) [reply docs/pages/desktop-access/troubleshooting.mdx:44]: I think it depends on whether or not you need to go down to that log level in order to enable them first or not. I guess I should try it for myself.

Comment (alexfornuto) [reply docs/pages/desktop-access/troubleshooting.mdx:44]: One does in fact need to traverse down the chain in order to enable the log.

Comment (stevenGravy) [reply docs/pages/desktop-access/troubleshooting.mdx]: @zmb3 ok with that?

File: docs/pages/desktop-access/troubleshooting.mdx [hunk 3]
```diff
@@ -39,8 +39,8 @@ or similar.
 
 **Solution:** Review the Security-Kerberos Log on the Windows host for causes.
 
-The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts.  This is available
-at `Event Viewer->Application and Services Logs->Microsoft->Windows->Security-Kerberos->Operational` in the Windows Event Log.  By default this
+The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts. This is available
```
Comment (ptgott) [inline docs/pages/desktop-access/troubleshooting.mdx:42]:
```suggestion
The Security-Kerberos Windows Event Log will provide information on Smart Card-based authentication attempts. This is available
```

**Review by ptgott — APPROVED** [review]
Approved with a minor grammatical suggestion.

**Pull request merged**